radix-engine/src/system/system_db_checker.rs

```diff
@@ -0,0 +1,746 @@
+use crate::system::node_modules::type_info::TypeInfoSubstate;
```

Comment:
Howe we move this file to `scrypto-tests`?

Reply:
I was slightly worried about this logic being used in the `src`, so it oversimplifies the system, although it's sufficient for testing purpose.

How about we add `#[cfg(test)]` to this file, if we want it in `radix-engine` instead of a tests folder?